app/index/components/navigationBox/navigation-box.import.less

```diff
@@ -0,0 +1,38 @@
+.index__navigation-box strong {
+	vertical-align: middle;
+	font-weight: bold;
+	font-size: 28px;
+	color: white;
+}
+.index__navigation-box {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	padding: 10px 50px;
+	height: auto;
+	width: auto;
+	margin: auto;
+	max-width: 1200px;
+}
+.index__navigation-box .index__navigation-box--images {
+	width: 320px;
+	height: 213px;
+	background-position: top center;
+	background-size: cover;
+	background-repeat: no-repeat;
+	margin: 0;
+	overflow: hidden;
+	border-top-right-radius: 10%;
+	border-top-left-radius: 10%;
```

Comment:
@davidshb19 I would change this to a `px` based value, just in case the size of the boxes change in the future. Not a big deal.

I also think they should be a little less rounded - 4px or so - to make them feel more modern.

Reply:
what do you mean by - 4px, do you mean negative 4